### ISSUE TITLE
COSYM: experiment identifiers need to be handled gracefully if working with > 1 integrated reflection file

### ISSUE THREAD

**Issue by graeme-winter** [dials/dials#657]
Related to https://github.com/dials/dials/issues/655 - if we have manually processed set A and set B they will both have experiment identifier "" and id 0 - ideally following #613 this would have been set to a unique hash rather than an empty string however would still have numerical ID 0.

This is probably easiest derived from #656 - however at the moment running cosym on 2 sets of (reflections, experiment) outputs one (reflection) and two (experiment) then subsequently 

```
Traceback (most recent call last):
  File "/Users/graeme/svn/cctbx/build/../modules/dials/command_line/symmetry.py", line 267, in <module>
    run(sys.argv[1:])
  File "/Users/graeme/svn/cctbx/build/../modules/dials/command_line/symmetry.py", line 130, in run
    assert len(experiments) == len(reflections)
AssertionError
```

**Comment by graeme-winter** [comment]
N.B. probably applies to symmetry as well.

**Comment by jbeilstenedmands** [comment]
This issue has been fixed by @rjgildea's recent commits c4800c1 and 0933897, which now handle this in the same way as dials.scale (this behaviour can be implemented at the module level after #686).
I'm therefore closing this issue but leaving the others related ones open.
